event-management-frontend/public/locales/ro/translation.json

```diff
@@ -0,0 +1,27 @@
+{
+  "eventList": {
+    "title": "Titlu",
+    "subtitle": "Subtitlu",
+    "status": "Status",
+    "active": "Activ",
+    "inactive": "Inactiv",
+    "notSet": "Nesetat",
```

Comment:
"Nesetat" nu exista in dex. Cred ca ar merge mai bine aici "nedeterminat"/ "nefixat". Putem merge si pe "Nu a fost setat", dar probabil se va strica putin stilizarea